app/common/lib/platformUtil.js

```diff
@@ -47,3 +50,16 @@ module.exports.isLinux = () => {
   return !module.exports.isDarwin() &&
     !module.exports.isWindows()
 }
+
+module.exports.formatOsPlatform = (os) => {
+  let platformMapping = {
+    'aix': 'AIX',
+    'darwin': 'macOS',
+    'freebsd': 'FreeBSD',
+    'linux': 'Linux',
+    'openbsd': 'OpenBSD',
+    'sunos': 'sunOS',
+    'win32': 'Microsoft Windows'
+  }
+  return platformMapping[os]
```

Comment:
This will be a good one ++. Can you just please handle null scenario as well. Let's say that os is not on our list we would like to return just os value as is and not undefined/null. Thank you